app/_src/guides/federate.md

```diff
@@ -0,0 +1,139 @@
+If you see `<Pending>` you either need to wait until load balancer is provisioned or you need to make sure that your Kubernetes cluster support provisioning load balancers. 
+
+## Copy resources from zone to global control plane
+
+To federate zone control plane without any traffic interruption, we need to copy resources like secrets, meshes etc.
+We can do this by executing the following command:
+
+```sh
+kubectl --context=mesh-zone port-forward svc/demo-app -n kuma-demo 5000:5000
+export ZONE_USER_ADMIN_TOKEN=$(kubectl --context=mesh-zone get secrets -n kuma-system admin-user-token -ojson | jq -r .data.value | base64 -d)
+kumactl config control-planes add \
+  --address http://localhost:5681 \
+  --headers "authorization=Bearer $ADMIN_TOKEN" \
+  --name "zone-cp" \
+  --overwrite  
+  
+kumactl export --profile=federation --format=kubernetes > resources.yaml
+```
+
+Apply resources on global control plane
+```sh
+kubectl apply --context=mesh-global -f resources.yaml
+```
+
+## Connect zone control plane to global control plane
+
+Update Helm deployment of zone control plane to configure connection to the global control plane.
+
+```sh
+helm upgrade --kube-context=mesh-zone --namespace {{site.mesh_namespace}} \
+--set {{site.set_flag_values_prefix}}controlPlane.mode=zone \
+--set {{site.set_flag_values_prefix}}controlPlane.zone=zone-1 \
+--set {{site.set_flag_values_prefix}}ingress.enabled=true
+--set {{site.set_flag_values_prefix}}controlPlane.kdsGlobalAddress=grpcs://<global-kds-address>:5685 \
+--set {{site.set_flag_values_prefix}}controlPlane.tls.kdsZoneClient.skipVerify=true \
+{{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+## Verify federation
+
+To verify federation, first port-forward the API service from the global control plane. 
+
+```sh
+kubectl --context=mesh-global port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 5681:5681
+```
+
+And then navigate to [127.0.0.1:5681/gui](http://127.0.0.1:5681/gui) to see the GUI.
+
+You should see a zone in list of zones as well as policies and all data plane proxies applied in the quickstart.
+
+### Apply policy on global control plane
+
+We can check policy synchronization from global control plane to zone control plane by applying a policy on global control plane:
+
+```
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshCircuitBreaker
+metadata:
+  name: demo-app-to-redis
+  namespace: {{site.mesh_namespace}}
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: demo-app_kuma-demo_svc_5000
+  to:
+  - targetRef:
+      kind: MeshService
+      name: redis_kuma-demo_svc_6379
+    default:
+      connectionLimits:
+        maxConnections: 2
+        maxPendingRequests: 8
+        maxRetries: 2
+        maxRequests: 2" | kubectl --context=mesh-global apply -f -
+```
+
+and verifying that it's synced to zone control plane
+```sh
+// TODO until we have a release 
```

Comment:
maybe they could just do 

```
kubectl get meshcircuitbreakers.kuma.io -A
```

and check for `demo-app-to-redis` ?

Reply:
yes, that's my plan, but it has suffix so I wanted to show the list with suffix hence a todo

Reply:
here you go:

```
$ kubectl get meshcircuitbreakers.kuma.io -A
NAMESPACE     NAME                                 TARGETREF KIND   TARGETREF NAME
kuma-system   mesh-circuit-breaker-all-default     Mesh
kuma-system   demo-app-to-redis-65xb45x2xfd5bf7f   MeshService      demo-app_kuma-demo_svc_5000
```